### DISCUSSION TITLE
Vercel Deploy Summary no longer works

### DISCUSSION THREAD

**Discussion by Stanzilla** [vercel/community#4024] [Bug Report]
### Page to Investigate

https://vercel.com/integrations/deploy-summary

### Steps to Reproduce

This integration no longer works. I tested it with several accounts and repositories, no PR comments get posted. I also pinged @leerob on Twitter about it.

**Comment by votemike** [comment]
Nothing working on my project either.
Paid Vercel account, recently created. Deploy Summary recently added.
